### ISSUE TITLE
Failure of getSourceAttachments method

### ISSUE THREAD

**Issue by stanben** [FamilySearch/familysearch-javascript-sdk#148]
I tried the getSourceAttachments using the person url returned for each person of the parent source object.
The problem I am observing is that in the cases where there is no person attached to that source person,  the getSourceAttachments method never returns. This causes my application to hang waiting for a response.  Shouldn't all SDK methods return with a response even if there is none? 

You can test this running  getSourceAttachments with  either of the following source person urls:
https://familysearch.org/pal:/MM9.1.1/QVJR-D2WM
https://familysearch.org/pal:/MM9.1.1/K8HX-54S

Both come from sources attached to R. Dean Benedict KWHX-CBB

**Comment by justincy** [comment]
It works properly for me. See this working example: http://jsfiddle.net/wef5hx7n/1/

Are you doing something different?

**Comment by stanben** [comment]
I am doing the same thing.
In order to keep working I created a list of 23 urls to ignore including
the one you tested.
I just made the change to stop ignoring those urls and they are now all
working correctly with getSourceAttachments returning with an empty list of
personIds as would be expected.
This must have been my error, but I am not sure what.

Thanks for the quick investigation.  You are welcome to close this issue.
